services/ui-src/src/types/users.ts

```diff
@@ -15,6 +15,7 @@ export interface MFPUser {
   full_name: string;
   state?: string;
   userRole?: string;
+  userReports?: string[] | undefined;
```

Comment:
will this ever be undefined if we are setting the [WP, SAR] array when reports is undefined?

Reply:
Don't need to add this because we're not restricting user report access for MFP.